cli/src/lib/libDefs.js

```diff
@@ -159,10 +159,24 @@ async function addLibDefs(pkgDirPath, libDefs: Array<LibDef>, validationErrs?: V
 /**
  * Given a 'definitions/npm' dir, return a list of LibDefs that it contains.
  */
+ const basePathRegex = /definitions\/npm\/[^\/]*\/?/;
 export async function getLibDefs(
```

Comment:
Rather than adding complexity to this general-purpose (and fairly fast) function, can we just filter the list of libdefs it returns in the `run-tests` command (and only do so during travis runs -- maybe an `--only-changed` flag or something)?

For example: `validate-defs` is very fast, so no good reason to make it not run on all libdefs (but the `run-tests` command is itself very slow, so only running relevant stuff there is a good idea).